niworkflows/viz/plots.py

```diff
@@ -206,6 +213,9 @@ def plot_carpet(
 
         data = img.get_fdata().T
         matrix = img.header.matrix
+
+        dilated_brainmask, _ = get_dilated_brainmask(atlaslabels, brainmask)
```

Comment:
why is this called again? don't we use the nipype interface above to wrap this function?

Reply:
At first glance, I'm not sure why I did call the function here. Let me try to implement it using the interface only.

Reply:
> why is this called again? don't we use the nipype interface above to wrap this function?

I did implement the changes necessary so that we compute the crown mask through the interface and not through the function call. Those changes implied modifying code in niworkflows (this PR), fmriprep (PR nipreps/fmriprep#2621) and mriqc (PR nipreps/mriqc#968).

Note that MRIQC PR is new (submitted last friday), as previous royal carpet plot implementation were only present in niworkflows with regards to MRIQC.

Furthermore for fmriprep, I took care that the crown mask is calculated only once in the `init_bold_confs_wf` workflow by transfering this variable to the `init_carpetplot_wf ` through its inputnode. (Previously the crown mask was calculated again in the latter workflow)